packages/core/scripts/collateral-umip/1_Propose.js

```diff
@@ -46,31 +58,50 @@ async function runExport() {
 
   const collaterals = _.castArray(argv.collateral);
   const fees = _.castArray(argv.fee);
-  const decimals = argv.decimals && _.castArray(argv.decimals);
-
-  if (collaterals.length !== fees.length || (decimals && decimals.length !== collaterals.length)) {
-    throw new Error("Must provide the same number of elements to --fee, --collateral, and --decimals (optional)");
+  const decimals = _.castArray(argv.decimals);
```

Comment:
If `argv.decimals === null` then `_.castArray(null, null, etc.)` returns [null, null, null] which I think is what the original intention was.